src/Tasks.UnitTests/AssignCulture_Tests.cs

```diff
@@ -216,6 +256,30 @@ public void PseudoLocalization(string culture)
             Assert.Equal($"MyResource.{culture}.resx", t.AssignedFiles[0].ItemSpec);
```

Comment:
This applies to the other asserts as well, we _should_ shouldly-ify functions we touch on.

```suggestion
            t.AssignedFiles[0].ItemSpec.ShouldBe($"MyResource.{culture}.resx");
```